velox/dwio/parquet/tests/reader/E2EFilterTest.cpp

```diff
@@ -60,7 +60,7 @@ class E2EFilterTest : public E2EFilterTestBase {
     sinkPtr_ = sink.get();
     options_.memoryPool = rootPool_.get();
 
-    options_.bufferGrowRatio = 2;
+    options_.bufferGrowRatio = 1.5;
```

Comment:
Do we have any test result for the performance improvement?

Reply:
@rui-mo Add the performance result and the root cause in the PR description.